# goal
geometry_msgs/TwistStamped nav_goal
---
# result
std_msgs/Bool outcome
---
# feedback
std_msgs/Char progress